### ISSUE TITLE
Personal Portfolio User Story #10 (Layout #2)

### ISSUE THREAD

**Issue by Amber-Pittman** [freeCodeCamp/testable-projects-fcc#406]
#### Issue Description

When I run a test in Full view, I pass all the user stories. However, whenever I'm in Editor view, I fail one user story 

> User Story #10: The height of the welcome section should be equal to the height of the viewport.

I realized I was behind on an update in Chrome when this form asked for Browser version. Thinking that might be the issue, I updated it. Unfortunately, I'm still having the same conundrum. 

Strangely enough, when I'm on mobile (using Chrome as well), I'm passing it in neither the Editor view nor the Full view. 

If I'm not passing it, I'm not passing it but if there's a chance that I am in fact passing User Story 10, I want to go ahead and submit it to completed projects in fCC. 

#### Browser Information

* Browser Name, Version: Chrome Version 67.0.3396.99 (Official Build) (64-bit)
* Operating System: Windows 10
* Mobile, Desktop, or Tablet: Desktop

#### Your Code / Link to Your Pen

[Amber Pittman's Personal Portfolio](https://codepen.io/amber-pittman/pen/pKZbNZ)

Thanks for any help or info. -Amber

**Comment by tbushman** [comment]
@Amber-Pittman Hello, you can change `min-height: 55vh;` to `height: 100vh;` to get your test to pass. `55vh` is 55% of the height of the viewport. That should fix it. :)

**Comment by Amber-Pittman** [comment]
Oh my gosh... I did that earlier and it didn't work. It wasn't until you
messaged me that I finally saw the "min" part of min-height. Lol sometimes
when things like that happen, I wonder if I'm a host from West World who's
been programmed to not see certain things until someone changes my
code. 😆 I'm truly glad you responded, because who knows how long it would
have taken me to notice it! Changing it did work, btw.

Thanks again, Tracey!

Amber
